### DISCUSSION TITLE
2023 RHDS Survey Feedback

### DISCUSSION THREAD

**Discussion by marionnegp** [RedHat-UX/red-hat-design-system#851] [Ideas]
We got some great feedback from our survey about the design system kit in XD and ux.redhat.com. Here are the suggestions categorized based on the amount of time or effort.

**Things we're already working on:**
- [ ] More dev documentation to help with implementation
	- Would this be solved with:
		- A new Development parent section with pages on ux dot
		- Better developer/code documentation on component pages
		- Different ways of using component code depending on the environment? (e.g. page builder vs. other websites)
- [ ] Code examples and sandboxes/playgrounds on ux.redhat.com
- [ ] Add “health status” or beta label for components that are in progress with link to GitHub issue
- [ ] Link to more things in GitHub to make contribution easier

**Things we could do quickly:**
- [ ] Add list of sites wholly or partially using design system
- [ ] Add guidance on where to find sources for design assets
- [ ] Add date when component was last updated to ux.redhat.com
- [ ] Add list of owners and contributors to ux.redhat.com

**Things that may take more time:**
- [ ] Demos, walkthroughs, and trainings
- [ ] More support for mobile mockup creation in the kit
- [ ] Search bar on ux.redhat.com
- [ ] More patterns and components with interaction states in the kit and/or ux dot
	- This might be solved by adding links to prototype preview of interaction states and behavior at different breakpoints
- [ ] Add guidelines for user research and testing
- [ ] Updated personalization kit

**Other feedback**
- Open source resources are hard to find
- There’s a need to make sure that RHDS guidance matches with Drupal output

**Comment by marionnegp** [comment]
We also got some quantitative data about frequency of use and satisfaction with the kit and ux dot.
![Screenshot 2023-03-24 at 12 08 31 PM](https://user-images.githubusercontent.com/95588923/227580418-e57e1ef0-d833-40df-ae9b-6eaa749aa4e8.png)
![Screenshot 2023-03-24 at 12 08 38 PM](https://user-images.githubusercontent.com/95588923/227580473-810bd94f-c6b5-4ba2-ad29-39cf71c62245.png)
![Screenshot 2023-03-24 at 12 08 46 PM](https://user-images.githubusercontent.com/95588923/227580493-48a84b86-fe38-406b-b436-ea99426a33b1.png)
![Screenshot 2023-03-24 at 12 08 50 PM](https://user-images.githubusercontent.com/95588923/227580508-75af7478-b236-4b0c-bf00-7d776d95bbec.png)

**Comment by bennypowers** [comment]
for search bar on ux-dot, I recommend we use pagefind, which provides a very nice client-side search experience. I had implemented this in the tokens branch - it turned out to not be the right fit there, but would be perfect for the general top-level site search. The tokens branch, or more specifically, the lack of a search-input component blocks this
